bbn-test-5/upgrades/v2/README.md

```diff
@@ -0,0 +1,30 @@
+# `v2` Software Upgrade
+
+## Upgrade overview
+
+- **Upgrade version**: `v2`
```

Comment:
Should this be v2.0.0-rc.0 ?

Reply:
https://github.com/babylonlabs-io/babylon/blob/v2.0.0-rc.0/app/upgrades/v2/upgrades.go#L31
The upgrade handler is named `v2` that is also used in the proposal plan name so I think we can use `v2` here